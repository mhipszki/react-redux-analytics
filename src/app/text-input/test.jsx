import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import TextInput from './';

test('styles a text input element', () => {
  const component = shallow(<TextInput />);
  expect(component).toMatchSnapshot();
});

test('has different styling when disabled', () => {
  const component = shallow(<TextInput disabled />);
  expect(component).toMatchSnapshot();
});
