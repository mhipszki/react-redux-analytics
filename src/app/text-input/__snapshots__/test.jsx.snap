// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has different styling when disabled 1`] = `
.c0 {
  background: #eee;
  border: none;
  border-radius: 3px;
  color: palevioletred;
  font-size: 1.5em;
  margin: 0.5em;
  padding: 0.5em;
}

<input
  className="c0"
  disabled={true}
  type="text"
/>
`;

exports[`styles a text input element 1`] = `
.c0 {
  background: papayawhip;
  border: none;
  border-radius: 3px;
  color: palevioletred;
  font-size: 1.5em;
  margin: 0.5em;
  padding: 0.5em;
}

<input
  className="c0"
  type="text"
/>
`;
